sos/plugins/ceph.py

```diff
@@ -50,28 +50,42 @@ def setup(self):
         ])
 
         self.add_cmd_output([
-            "ceph status",
-            "ceph health detail",
-            "ceph osd tree",
-            "ceph osd stat",
-            "ceph osd dump",
             "ceph mon stat",
             "ceph mon_status",
             "ceph quorum_status",
-            "ceph mon dump",
-            "ceph df",
+            "ceph osd erasure-code-profile ls",
             "ceph report",
-            "ceph osd df tree",
-            "ceph fs dump --format json-pretty",
-            "ceph fs ls",
-            "ceph pg dump",
-            "ceph health detail --format json-pretty",
             "ceph osd crush show-tunables",
             "ceph-disk list",
             "ceph versions",
             "ceph osd crush dump"
+            "ceph -v"
         ])
 
+        ceph_cmds = [
+            "status",
+            "health detail",
+            "osd tree",
+            "osd stat",
+            "osd df tree",
+            "osd dump",
+            "osd df",
+            "mon dump",
+            "df",
+            "df detail",
+            "fs ls",
+            "fs dump",
+            "pg dump",
+        ]
```

Comment:
Why some `ceph *` commands are taken from this list while others are executed "directly" (cf. L58-62 and L82)? Just because the later ones are taken with json format as well? (maybe renaming the `ceph_cmds` to something more obvious will help code clarity)

Reply:
Because those commands share both regular execution:
```
        ceph_cmds = [
            "status",
            "health detail",
            "osd tree",
            "osd stat",
            "osd df tree",
            "osd dump",
            "osd df",
            "mon dump",
            "df",
            "df detail",
            "fs ls",
            "fs dump",
            "pg dump",
        ]

        self.add_cmd_output([
            "ceph %s" % s for s in ceph_cmds
        ])

```

As well the same command can be re-used and collect json-pretty

```
        self.add_cmd_output([
            "ceph %s --format json-pretty" % s for s in ceph_cmds
        ], subdir="json_output")
```

Reply:
That output is using same commands from ceph_cmds, they are shared and just subdirectory is different, this is to avoid duplicated code.